report/Introduction/ProjectProposal_eng.tex

```diff
@@ -0,0 +1,58 @@
+This gives the designer some insight into what the customer actually wants and likes. 
+\\\\
+With these two problems concerning inclusion of the customer in the development process, it would be exiting to look at which other technologies could be used, that could allow the customer to express their needs while being in their problem domain while still being able to communicate with the developers. 
+
+\subsection{Execution}
+We would like to create a mobile application which will allow the customer to express their needs in an intuitive and precise manner, which at the same time would allow the requirements to be translated to a set of "acceptance tests" which would allow the developers to verify that they have fulfilled the requirements of the customer.
+This tool will also allow the developers to propose alternative solutions, not in the form of software implementations, but in the same environment as the customer. 
+This will allow the customer to look at the different problem solutions whenever they like to, and all in the same environment that the customer uses to express their requirements. 
+\\\\
+The intention is to develop a platform that will use "Finite State Automata" to express needs, requirements and wishes to a product, in a way that is useful for people without formal knowledge of these. 
+
+Using "Finite State Automata" will also allow for algorithms to highlight errors or oversights in the specified requirements.
+The use of "Finite State Automata" will also make it possible to know every state of an application, and this means that it is possible to give guarantees that applications will not end in a unwanted state. 
+
+Furthermore this would also allow that developers with far greater knowledge about "Finite State Automata" to communicate alternative solutions in a way that is transparent and understandable to the customer. 
+
+These State Automatas will then be translated into acceptance tests though automatic code generation.
+
+This will in the end allow the developer greater confidence in developing a product that fulfils the customers wishes, while also make it easier to get the development right the first time.
+\\\\
+We would like during the project to perform an experiment, with the proposed solution to test if the product could solve some of the problems.
+To do this we would like to select some of the groups from the Giraf proect at Aalborg Universitet. 
+Here we would act as a communication medium between the development teams (groups) and Product Owner.
+
+Here we will start by creating paper prototypes of our idea, and see how communication and understanding will be affected by both customer (Product Owner) and developers (groups).
+
+After this we would have to manually rewrite the paper prototype to "acceptance tests" in the project and see how it would be to have automatic "Definition of Done" affects developers ability to fulfil the requirements
```

Comment:
```suggestion
After this we would have to manually rewrite the paper prototype to "acceptance tests" in the project and see how it would be to have automatic "Definition of Done" affects developers ability to fulfill the requirements
```